### ISSUE TITLE
Needs to be converted to Process API from the deprecated WirePattern

### ISSUE THREAD

**Issue by linonetwo** [noflo/noflo-image#239]
node_modules\noflo-image\components\CalculateAspectRatio.coffee:18 

Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'WirePattern')
    at exports.getComponent (E:\repo\TiddlyGit-Desktop\node_modules\.pnpm\noflo-image@0.4.0\node_modules\noflo-image\components\CalculateAspectRatio.coffee:18:17)

```coffeescript
  noflo.helpers.WirePattern c,
```

**Comment by linonetwo** [comment]
## 1.2.0 (August 28th 2020)

* Removed support for `noflo.WirePattern`. WirePattern has been deprecated since 1.0, and all code using it should be migrated to the latest Process API

**Comment by bergie** [comment]
Exactly, WirePattern has been deprecated for six years now. Process API is the way to go nowadays.

https://bergie.iki.fi/blog/noflo-process-api/

**Comment by linonetwo** [comment]
Wait, you can't close this issue as completed, because it make this repo unuseable with latest noflo.

**Comment by bergie** [comment]
Ok, relabelling.
